sdk/storage/storage-blob/src/BlobLeaseClient.ts

```diff
@@ -0,0 +1,419 @@
+import { generateUuid, HttpResponse } from "@azure/core-http";
+import { StorageClientContext } from "./generated/src/storageClient";
+import { ContainerBreakLeaseOptionalParams } from "./generatedModels";
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why lease is affected, BTW the copyright format is strange here. should it on the top?

Reply:
fixed